schema/newschma.yaml

```diff
@@ -0,0 +1,32 @@
+root:
+  parts: # We dropped the SEQUENCE information from the excel sheet
+   name: string
+   role: enum
+   circular: boolean # isn't this implicit from role ?
+   partDescription: string
+   application: 
+    applicationNotes: string # formerly "Design Notes"
+    targetOrganism: string
+    finalProduct: string # Should we keep this? Where should we keep this?
```

Comment:
"finalProduct" is distribution information.

Reply:
I'm trying to recollect if this should be under `application` or not.

Reply:
From what you said earlier, I think this was a misclassification from interpreting "product" as "thing made by biology, like a protein or a gRNA", rather than as "thing in a well on a distribution plate"